docs/_posts/2019-12-31-ace3-version3130.md

```diff
@@ -0,0 +1,115 @@
+---
+title: ACEREP  &#35;00010
+description: Status report on ACE3 version 3.13.0
+parent: posts
+image: /img/news/191231_medical1.jpg
+author: alganthe, mharis001
+layout: post
+---
+
+This post marks the release of ACE3 v3.13.0 and, as a result, the public release of the medical rewrite. The previous news post provided perspective on the guiding principles and goals of the medical rewrite. In this post, we will detail the key changes of the rewrite in order to give users a better starting point as they explore ACE3's new medical system.
+
+<!--more-->
+
```

Comment:
```suggestion

Medical rewrite was also as the name suggests, a complete rewrite of the medical system. Most of the changes however will not be noticeable by the users, however they provide an improved code base for easier and faster development of the system in the future, opening possibilities for more features.

```
I would add something like that somewhere, doesn't have to be here or in that form.